stripe/api_resources/abstract/test_helpers.py

```diff
@@ -6,9 +6,7 @@
 from typing import TypeVar, ClassVar, Any
 from typing_extensions import Protocol
 
-from stripe.api_resources.abstract.api_resource import APIResource
-
-T = TypeVar("T", bound=APIResource)
```

Comment:
Did the bound not work out?

Reply:
Sort of. The trouble is that APIResource needs a parameter but I guess `bound=APIResource[Any]` is good enough.

Stylistically, I consider unused bounds on type parameters to be a code smell, but in this case the only reason it's "unused" is because we can't properly type things and have that `_resource_cls: ClassVar[Any]` business, so I changed my mind.

Reply:
(Fixed)